frontend/src/features/rollout-announcement/index.ts

```diff
@@ -0,0 +1 @@
+export { RolloutAnnouncementModal as default } from './RolloutAnnouncementModal'
```

Comment:
prefer non-default exports, read https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad?gi=4bb93b11b

the few default exports we have in the codebase is due to dynamic imports for chunking, which this component does not need

Reply:
Ahh okay thanks, I updated to use non-default export